config/locales/de.yml

```diff
@@ -254,6 +254,7 @@ de:
         <<: *id-generic_keys
         amount: Betrag
         assignment: Begleitung
+        created_by: Verantwortliche/r
```

Comment:
where is this still used? In general for created_by should be used "angemeldet von"

Reply:
At billing expenses

Reply:
ah true!